docs/_docs/variables.md

```diff
@@ -15,33 +16,48 @@ nav_order: 11
 - Text weight
 - Corner radius
 
-Using variables instead of fixed values in your Figma design file requires no change on the developer side. In order to change a variable value in your application, DesignCompose has several functions to help with this.
+In order to change a variable value in your application, DesignCompose has several functions to help
+with this.
+
+However, developers must have a full seat in an **Enterprise** org to use the [Variables REST
+API][3]. If a design file uses variables, developers need to use a [figma auth token][4] that
+has the `file_variables:read` scope to be able to download the variables. If developers are not
+part of Enterprise plan or use an auth token that don't have access to variables, the design will
+use the node values directly and the variable functions, as described below, will not change upon
+changing variable collection or variable mode.
```

Comment:
I would say something like: "use the node values directly, and the variable functions below will have no effect."

Reply:
Done.